# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/extensions/webidl-api/ExtensionBrowser.cpp
# Commit: 5598943a94fd
# Full Hash: 5598943a94fd9e6be45bbc7125ef7bac49a4bf05
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-12-15 21:51:13
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Depends on D132708
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r 43186fbbf8b4 -r 5598943a94fd toolkit/components/extensions/webidl-api/ExtensionBrowser.cpp
--- a/toolkit/components/extensions/webidl-api/ExtensionBrowser.cpp	Wed Dec 15 13:48:17 2021 +0000
+++ b/toolkit/components/extensions/webidl-api/ExtensionBrowser.cpp	Wed Dec 15 13:48:17 2021 +0000
@@ -8,7 +8,7 @@
 
 #include "mozilla/dom/ExtensionBrowserBinding.h"
 #include "mozilla/dom/ExtensionPortBinding.h"  // ExtensionPortDescriptor
-#include "mozilla/dom/WorkerPrivate.h"         // GetWorkerPrivateFromContext
+#include "mozilla/dom/WorkerScope.h"           // GetWorkerPrivateFromContext
 #include "mozilla/extensions/ExtensionAlarms.h"
 #include "mozilla/extensions/ExtensionMockAPI.h"
 #include "mozilla/extensions/ExtensionPort.h"
